### ISSUE TITLE
[Question] Use instructions

### ISSUE THREAD

**Issue by cassianotartari** [Phobetor/rabbitmq-supervisor-bundle#11]
Hi!

   First time playing with Supervisor and I've installed this bundle to generate conf files. Seems to work great. But I have some silly questions:
- The `pid` and `sock` files are generated just when Supervisor is running? That's because in the `Zero Configuration` section is showing these files in the tree structure and after run `bin/console rabbitmq-supervisor:rebuild` both files weren't generated.
- Should I override `/etc/supervisor/supervisord.conf` with `app/supervisor/dev/supervisord.conf` and reload supervisor service (`supervisorctl reread && supervisorctl update`) to get built conf files running?
  
  I'm running all in a vagrant Homestead virtual machine under Symfony 3.1.1.

Thanks!

**Comment by cassianotartari** [comment]
Digging here... seems to be related to startup script under ``/etc/init.d``. I've installed supervisor by ``sudo apt-get install supervisor``. Looking under ``/etc/init.d/supervisor`` (automatically generated) has in the beginning:
```bash
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
DAEMON=/usr/bin/supervisord
NAME=supervisord
DESC=supervisor

test -x $DAEMON || exit 0

LOGDIR=/var/log/supervisor
PIDFILE=/var/run/$NAME.pid
```
Am I saying stupid? I've already tried just copy conf workers files to ``/etc/supervisor/conf.d`` keeping original ``/etc/supervisor/supervisord.conf`` but when I run ``sudo supervisorctl`` it throws ``unix:///var/run/supervisor.sock no such file``. Just work when I do ``sudo service supervisord restart``.

Any ideas? Maybe this is not the right place for this issue.
